### PR TITLE
feat(hud,controller): fuse hud into controller for input handling

### PR DESCRIPTION
**Pull request by Polostor**
Based on [discussion](https://github.com/DigitalExtinction/Game/pull/317#discussion_r1070443664) while doing new HUD without circular coupling of hud and controller crates.

### CODE REVIEW

**Review by Indy2222 — APPROVED** [review]

**Review by Indy2222 — CHANGES_REQUESTED** [review]
Please delete all mentions of the now removed crate from CONTRIBUTING.md.

**Comment by Polostor** [comment]
While changing `CONTRIBUTING.md` I saw a discrepancy in `Topologically sorted crates:`.. Some have `de` prefix and others do not. @Indy2222 Is that intentional?

**Comment by Indy2222** [comment]
> While changing `CONTRIBUTING.md` I saw a discrepancy in `Topologically sorted crates:`.. Some have `de_` prefix and others do not. @Indy2222 Is that intentional?

No, feel free to fix that.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -126,9 +126,7 @@ Topologically sorted crates:
 * [combat](/crates/combat) – attacking, projectile & laser simulation and
   similar.
 
-* [hud](/crates/hud) – head-up display with 2D in-game UI.
-
-* [controller](/crates/controller) – handling of user input.
+* [controller](/crates/controller) – handling of user input and containing head-up display with 2D in-game UI.
```
Comment (Indy2222) [inline CONTRIBUTING.md:129]: Please break lines at 79 character threshold.

**Review by Indy2222 — CHANGES_REQUESTED** [review]
Only small-ish notes. Thank you!

Whenever possible:

* keep first commit message line <= 50 characters & use imperative
* write all additional info to the commit description (e.g. issue references, context, reasoning, etc.)

**Review by Indy2222 — APPROVED** [review]

**Pull request merged**